app/views/transportMeans/departure/MeansIdentificationNumberView.scala.html

```diff
@@ -18,23 +18,24 @@
 @import controllers.transportMeans.departure.routes
 @import models.LocalReferenceNumber
 @import viewModels.components.InputTextViewModel._
-@import views.html.components.{Button, ErrorSummary, InputText}
+@import views.html.components.{Button, ErrorSummary, InputText, InsetText}
 @import models.transportMeans.departure.Identification
 
 @this(
     mainTemplate: MainTemplate,
     formHelper: FormWithCSRF,
     inputText: InputText,
+    insetText: InsetText,
     errorSummary: ErrorSummary,
     button: Button,
     phaseConfig: PhaseConfig
 )
 
 
-@(form: Form[String], lrn: LocalReferenceNumber, mode: Mode, prefix: String, args: String*)(implicit request: Request[_], messages: Messages)
+@(form: Form[String], lrn: LocalReferenceNumber, mode: Mode, prefix: String, identificationNumber: String)(implicit request: Request[_], messages: Messages)
```

Comment:
Looks like we don't need to pass `prefix` in anymore and can just define it statically in the controller and view as is our standard pattern